src/planner/plan/append_scan_node_table.cpp

```diff
@@ -25,7 +25,9 @@ void Planner::appendScanNodeTable(std::shared_ptr<Expression> nodeID,
     auto scan = make_shared<LogicalScanNodeTable>(std::move(nodeID), std::move(tableIDs),
         propertiesToScan_);
     scan->computeFactorizedSchema();
-    plan.setCardinality(cardinalityEstimator.estimateScanNode(scan.get()));
+    // TODO(Guodong): Should remove cardinality from Plan.
```

Comment:
should this TODO be resolved in this PR?

Reply:
probably too many changes to do in this PR, will leave the TODO

Reply:
Yeah. Let's keep it as a refactoring but separate the changes in a following PR.